### ISSUE TITLE
Implementation on PyTorch

### ISSUE THREAD

**Issue by wlgns0105** [MatthieuCourbariaux/BinaryNet#23]
I'm trying to implement BinaryNet on PyTorch and trying to train on CIFAR-10 dataset, I have some questions

1. Is BNN (Binarized Neural Networks) same as BinaryNet ?

both paper writes their CIFAR-10 test error rate as 11.40% and BNN's code link goes to BinaryNet

2. On BinaryNet paper, it says no data augmentation was used, so is it correct that there's only
image data [0,255] (int) scaled & shifted to [-1,1] (float) ? 

in other words, no crop/brighten/resize etc. ?
I don't understand how original CNN (10.96%) and Binarized CNN (11.40%) have same error rate...

**Comment by MatthieuCourbariaux** [comment]
1. Yes. Those are just different versions of the same paper.
The 2 different error rates correspond to 2 different implementations (1 in Theano, 1 in Torch) of the same method. This git repository contains the Theano implementation. The Torch implementation is available [here](https://github.com/itayhubara/BinaryNet).

2. No data augmentation is used. There is only the preprocessed image dataset. 
If I recall correctly, we do some normalization and ZCA whitening.

**Comment by wlgns0105** [comment]
Thank you for the reply
